cfgov/v1/models/base.py

```diff
@@ -49,6 +49,18 @@ class Meta:
         verbose_name = _("Tag")
         verbose_name_plural = _("Tags")
 
+class CFGOVContentOwner(TagBase):
+    content_object = ParentalKey('CFGOVPage')
+
```

Comment:
```suggestion
```

@csebianlander I think this line is the problem -- compare against [the example in the Wagtail docs](https://docs.wagtail.org/en/stable/reference/pages/model_recipes.html#custom-tag-models) and you'll see that specifying the `content_object` is needed not on the tag class itself but on the "through model". I tried deleting this and recreating the migration and seem to be able to set the content owners okay, with autocomplete working as expected:

![image](https://user-images.githubusercontent.com/654645/156592893-3a39ea33-1162-41b5-abc2-67d98b1026fe.png)

Reply:
Ahh! I'd added that in at some point in attempting to debug and forgot to remove it to try to maintain the structure of the original code. This works great! Will re-commit and turn this back into a proper PR once I get lint and the other tests to pass locally.